configure_task.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+## Usage: ./camunda-deployment [SERVICE_TOKEN]
+##
+## Options:
+##    - SERVICE_TOKEN: a service token for a whitelisted service in camunda which is generated with the idam-service-token.
+##
+
+microservice="${1:-wa_task_configuration_api}"
+
+serviceToken=$(curl --silent --show-error -X POST \
+  -H "Content-Type: application/json" \
+  -d '{"microservice":"'${microservice}'"}' \
+  ${S2S_URL}/testing-support/lease
+)
+
+echo $serviceToken
+
+# GET TASKS
+response=$(curl --request POST \
+  --url http://camunda-api-aat.service.core-compute-aat.internal/engine-rest/task \
+  --header 'Content-Type: application/json' \
+  --cookie dtCookie=v_4_srv_2_sn_708A526309E3A599F638913244C50A40_perc_100000_ol_0_mul_1_app-3Aea7c4b59f27d43eb_1_rcs-3Acss_0 \
+  --data '{
+    "orQueries": [
+        {
+            "processVariables": [
+                {
+                    "name": "taskState",
+                    "operator": "eq",
+                    "value": "unconfigured"
+                }
+            ]
+        }
+    ],
```

Comment:
I think we should also filter on the Task's age.  Otherwise you could start interfering with Tasks that are _currently_ being configured and try to configure them again.  So maybe the query can look for Tasks that are > 1 min old?  That should give enough time.